Dockerfile

```diff
@@ -13,19 +14,42 @@ LABEL org.opencontainers.image.created=$BUILD_DATE \
   org.opencontainers.image.schema-version="1.0.0-rc.1" \
   org.opencontainers.image.license="MIT"
 
-COPY sh/clean.sh /clean.sh
+SHELL ["/bin/ash", "-euo", "pipefail", "-c"]
 
-RUN apk add --no-cache mariadb=10.4.18-r0 && \
-  /bin/sh /clean.sh && \
-  # removed in cleaning
+RUN \
+  apk add --no-cache mariadb=10.4.18-r0 && \
+  TO_KEEP=$(echo " \
+    usr/bin/mysql$ \
+    usr/bin/mysqld$ \
+    usr/bin/mariadb$ \
+    usr/bin/getconf$ \
+    usr/bin/getent$ \
+    usr/bin/my_print_defaults$ \
+    usr/bin/mysql_install_db$ \
+    usr/share/mariadb/charsets \
+    usr/share/mariadb/english \
+    usr/share/mariadb/mysql_system_tables.sql$ \
+    usr/share/mariadb/mysql_performance_tables.sql$ \
+    usr/share/mariadb/mysql_system_tables_data.sql$ \
+    usr/share/mariadb/maria_add_gis_sp_bootstrap.sql$ \
+    usr/share/mariadb/fill_help_tables.sql$" | \
+    tr -d " \t\n\r" | sed -e 's/usr/|usr/g' -e 's/^.//') && \
+  INSTALLED_FILES="$(apk info -q -L mariadb-client | tail -n +2) \
+  $(apk info -q -L mariadb-common | tail -n +2) \
+  $(apk info -q -L mariadb | tail -n +2)" && \
+  for path in $(echo "${INSTALLED_FILES}" | grep -v -E "${TO_KEEP}"); do \
+    eval rm -rf "${path}"; \
+  done && \
   touch /usr/share/mariadb/mysql_test_db.sql && \
   # allow anyone to connect by default
   sed -i -e 's/127.0.0.1/%/' /usr/share/mariadb/mysql_system_tables_data.sql && \
   mkdir /run/mysqld && \
   chown mysql:mysql /etc/my.cnf.d/ /run/mysqld /usr/share/mariadb/mysql_system_tables_data.sql
 
+# The one installed by MariaDB was removed in the clean step above, the size was to significant
```

Comment:
```suggestion
# The one installed by MariaDB was removed in the clean step above due to its large footprint
```

Reply:
Can you incorporate this to your commit? Not sure how I can do that from my end. Will land after!